components/body-map.js

```diff
@@ -118,23 +135,25 @@ export function BodyMap({
               organFilters={organFilters}
               enabledBodyMapFilters={enabledBodyMapFilters}
               handleClickOrgan={handleClickOrgan}
+              highlightedOrgans={highlightedOrgans}
+              highlightOrgans={highlightOrgans}
+              setHighlightedOrgans={setHighlightedOrgans}
             />
             <div>
               <ul className="list-disc grid grid-cols-2 gap-2">
                 {organList.map((organ) => {
                   const isSelected = organFilters.includes(organ);
                   const isDisabled = !enabledBodyMapFilters.includes(organ);
+                  const isHighlighted = highlightedOrgans.includes(organ);
                   return (
                     <li key={`${organ}-bodymap-organlist`}>
                       <button
                         id={organ}
                         role="button"
                         disabled={isDisabled}
-                        className={`hover:bg-slate-200 ${
-                          isDisabled && "text-slate-400"
-                        } ${
+                        className={`${isDisabled && "text-slate-400"} ${
```

Comment:
I think this generates `class="false"` in the generated HTML if `isDisabled`, `isSelected`, or `isHighlighted` has the value "false." This causes no apparent problems, but it doesn’t seem proper to use class names that don’t exist. Better to use `isDisabled ? "text-slate-400" : ""` instead. Might be best to assign these to variables before rendering, and use those variables here, because these expressions are pretty long.